test/output/openapi-to-asyncapi/operation_and_parameter_client.yml

```diff
@@ -0,0 +1,162 @@
+asyncapi: 3.0.0
+info:
+  title: Operations and Parameters API
+  version: 1.0.0
+  description: An API showcasing various operations and parameter types
+servers:
+  example_com_v1:
+    host: api.example.com
+    pathname: /v1
+    protocol: https
+channels:
+  items:
+    address: /items
+    messages:
+      listItemsResponse200:
+        name: listItemsResponse200
+        title: GET response 200
+        contentType: application/json
+        payload:
+          type: array
+          items:
+            $ref: '#/components/schemas/Item'
+        summary: Successful response
+      createItemRequest:
+        name: createItemRequest
+        title: POST request
+        contentType: application/json
+        payload:
+          $ref: '#/components/schemas/ItemInput'
+      createItemResponse201:
+        name: createItemResponse201
+        title: POST response 201
+        contentType: application/json
+        payload:
+          $ref: '#/components/schemas/Item'
+        summary: Created
+    parameters:
+      limit:
+        description: Maximum number of items to return
+        location: $message.header#/limit
+      offset:
+        description: Number of items to skip
+        location: $message.header#/offset
+  'items_{itemId}':
+    address: '/items/{itemId}'
+    messages:
+      getItemResponse200:
+        name: getItemResponse200
+        title: GET response 200
+        contentType: application/json
+        payload:
+          $ref: '#/components/schemas/Item'
+        summary: Successful response
+      updateItemRequest:
+        name: updateItemRequest
+        title: PUT request
+        contentType: application/json
+        payload:
+          $ref: '#/components/schemas/ItemInput'
+      updateItemResponse200:
+        name: updateItemResponse200
+        title: PUT response 200
+        contentType: application/json
+        payload:
+          $ref: '#/components/schemas/Item'
+        summary: Successful response
+      deleteItemResponse204:
+        name: deleteItemResponse204
+        title: DELETE response 204
+        summary: Successful response
+operations:
+  listItems:
+    action: send
+    channel:
+      $ref: '#/channels/items'
+    summary: List items
+    bindings:
+      http:
+        method: GET
```

Comment:
Not sure what to do here. Technically by not defining `messages`, all messages within `#/channels/items` is assumed to be possible...

I think we might have to define an empty list here 🤔

Reply:
I have define an empty list when there are no specific `messages` in `operation`

Reply:
I have define an empty list when there are no specific `messages` in operation